From d7bd19d7f5eaa862d9ba06c591e65bb4a1cd4462 Mon Sep 17 00:00:00 2001
From: "Chen, Haochuan" <haochuan.z.chen@intel.com>
Date: Thu, 28 Dec 2023 18:25:47 +0800
Subject: [PATCH] Enable LCG audio playback with WA

Call context routing to device media_out

Tests done:
- launch LCG with audio playback

Tracked-On: OAM-114407
Change-Id: I0da680ab6ea3a9bdb44d24700f5aa441d00e32a8
Signed-off-by: Chen, Haochuan <haochuan.z.chen@intel.com>
---
 .../default/policy/car_audio_configuration.xml                | 4 +---
 .../overlay-multi_passenger/audio/car_audio_configuration.xml | 4 +---
 2 files changed, 2 insertions(+), 6 deletions(-)

diff --git a/groups/audio/project-celadon/default/policy/car_audio_configuration.xml b/groups/audio/project-celadon/default/policy/car_audio_configuration.xml
index 99b474a..be36074 100644
--- a/groups/audio/project-celadon/default/policy/car_audio_configuration.xml
+++ b/groups/audio/project-celadon/default/policy/car_audio_configuration.xml
@@ -29,6 +29,7 @@
                     <device address="bus0_media_out">
                         <context context="music"/>
                         <context context="announcement"/>
+                        <context context="call"/>
                     </device>
                     <device address="bus6_notification_out">
                         <context context="notification"/>
@@ -43,9 +44,6 @@
                     </device>
                 </group>
                 <group>
-                    <device address="bus4_call_out">
-                        <context context="call"/>
-                    </device>
                     <device address="bus3_call_ring_out">
                         <context context="call_ring"/>
                     </device>
diff --git a/groups/multi-passenger/true/overlay-multi_passenger/audio/car_audio_configuration.xml b/groups/multi-passenger/true/overlay-multi_passenger/audio/car_audio_configuration.xml
index 14ff54a..8aed2c5 100644
--- a/groups/multi-passenger/true/overlay-multi_passenger/audio/car_audio_configuration.xml
+++ b/groups/multi-passenger/true/overlay-multi_passenger/audio/car_audio_configuration.xml
@@ -29,6 +29,7 @@
                     <device address="bus0_media_out">
                         <context context="music"/>
                         <context context="announcement"/>
+                        <context context="call"/>
                     </device>
                     <device address="bus6_notification_out">
                         <context context="notification"/>
@@ -43,9 +44,6 @@
                     </device>
                 </group>
                 <group>
-                    <device address="bus4_call_out">
-                        <context context="call"/>
-                    </device>
                     <device address="bus3_call_ring_out">
                         <context context="call_ring"/>
                     </device>
-- 
2.34.1

